### PR TITLE
JBPM-5355 - Support case project - adding Milestone service task to d…

### PR DESCRIPTION
**Pull request by tsurdilo**
…efault wid with its default icon

### CODE REVIEW

File: jbpm-designer-client/src/main/resources/org/jbpm/designer/public/defaults/WorkDefinitions.wid.st [hunk 1]
```diff
@@ -56,6 +56,15 @@ import org.drools.core.process.core.datatype.impl.type.ObjectDataType;
...
     "icon" : "defaultservicenodeicon.png"
+  ],
+
+  [
+    "name" : "Milestone",
+    "parameters" : [
+      "Condition" : new StringDataType()
+    ],
+    "displayName" : "Milestone",
+    "icon" : "defaultmilestoneicon.png"
```
Comment (mswiderski) [inline jbpm-designer-client/src/main/resources/org/jbpm/designer/public/defaults/WorkDefinitions.wid.st]: just one comment - not sure milestone should be added to all none case projects as that's what we do when console creates new case project to provide special types there instead. But if we see a point in having it by default in all projects that's ok to.

**Comment by mswiderski** [comment]
looks good to me - added one comment/question
+1

**Comment by tsurdilo** [comment]
@mswiderski amended

**Comment by mswiderski** [comment]
perfect, ready to be merged

**Pull request merged**